pkg/controller/reconciler/deployment_reconciler_test.go

```diff
@@ -42,6 +42,10 @@ const (
 	deploymentTestWaitDuration            = time.Second * 10
 	deploymentTestWaitInterval            = time.Second * 1
 	deploymentTestControllerDelayInterval = time.Millisecond * 100
+
+	fetchEventWaitDuration            = time.Second * 10
+	fetchEventWaitInterval            = time.Second * 1
+	fetchEventControllerDelayInterval = time.Millisecond * 100
```

Comment:
nit: I think you could just use `deploymentTestWaitDuration` and `deploymentTestWaitInterval`. They really have the same purpose.

Reply:
`fetchEventControllerDelayInterval` looks unused, can you remove it?

Reply:
Done